RecoTracker/IterativeTracking/python/TobTecStep_cff.py

```diff
@@ -21,7 +21,7 @@
 tobTecStepSeedLayersTripl = cms.EDProducer("SeedingLayersEDProducer",
     layerList = cms.vstring(
     #TOB
-    'TOB1+TOB2+MTOB3',
+    'TOB1+TOB2+MTOB3','TOB1+TOB2+MTOB4',
```

Comment:
how it when in here?
I stashed before commit!
will remove!
still quite disappointed by git behaviour...